ui.apps/src/main/content/jcr_root/apps/etoolbox-authoring-kit/allowed-children/js.txt

```diff
@@ -0,0 +1 @@
+js/allowedChildren.js
```

Comment:
(cosmetic) Why is the file not named in the same style as the folder?)

Reply:
File is named in the same way as dependsOn files